networkx/algorithms/threshold.py

```diff
@@ -32,22 +29,30 @@ def is_threshold_graph(G):
     >>> is_threshold_graph(G)
     False
 
-    References
-    ----------
-    .. [1] Threshold graphs: https://en.wikipedia.org/wiki/Threshold_graph
+    Returns
+    -------
+    True if `G` is a threshold graph, `False` otherwise.
     """
     return is_threshold_sequence(list(d for n, d in G.degree()))
 
 
 def is_threshold_sequence(degree_sequence):
-    """
-    Returns True if the sequence is a threshold degree seqeunce.
+    """Returns True if the sequence is a threshold degree seqeunce.
```

Comment:
This typo was already around, but might as well fix it while we're at it!
```suggestion
    """Returns True if the sequence is a threshold degree sequence.
```